string address
robonomics_msgs/Multihash model
robonomics_msgs/Multihash objective
string result
string promisee
string promisor
string lighthouse
string token
uint64 cost
string validator
uint64 validatorFee
